src/test/java/com/traveltime/sdk/TimeFilterFastProtoTest.java

```diff
@@ -43,4 +45,20 @@ public void shouldSendTimeFilterProtoRequest() {
         Either<TravelTimeError, TimeFilterFastProtoResponse> response = sdk.sendProto(request);
         Assert.assertTrue(response.isRight());
     }
+
+    @Test
+    public void shouldSendAsyncTimeFilterProtoRequest() throws ExecutionException, InterruptedException {
```

Comment:
When doing `async` operations, I would expect test when sending multiple requests at the same time. So I would add additional test.

Reply:
Sure, I'll add such test